demo/bgp/main.go

```diff
@@ -70,7 +70,7 @@ func CreateDemoBGPInstance() (*oc.Bgp, error) {
 	// The returned struct represents the new peer that was created.
 	nPeer.PeerAs = ygot.Uint32(29636)
 	nPeer.Description = ygot.String("catalyst2 Services Ltd")
-	nPeer.PeerType = oc.OpenconfigBgp_PeerType_INTERNAL
+	nPeer.PeerType = oc.OpenconfigBgpTypes_PeerType_INTERNAL
```

Comment:
I think I misunderstood something in our implementation strategy here. 

AIUI, this should only occur when we've changed the flag to change the names? Since this is a backwards incompatible change then I think we want it to be flag-protected -- but it looks like it's being done by default here without a flag.

Reply:
I will add the flags to help with the transition, but I'll assume the example files will change since they should showcase best practice.

Reply:
Yes, I think that'll be the case -- but this looks difficult for users to consume otherwise.